docs/training/apis-howto/api-core-ug.rst

```diff
@@ -1,212 +1,302 @@
-##########
- Core API
-##########
+#####################
+ Core API User Guide
+#####################
 
-In this guide, you'll learn how to use the Core API.
```

Comment:
Are we calling it "Core API" or "the Core API"? Thought it was the former

Reply:
I don't care, @tara-det-ai likes "Core API".  Sure.

Reply:
throughout the docs, "the Core API" is most common and is more grammatically correct

Reply:
resolved